webpack.config.js

```diff
@@ -1,11 +1,42 @@
 const path = require('path');
 const outputDir = path.resolve(__dirname, './dist');
 
+const {SchemaToDefaultsPlugin} = require('openfin-service-config/plugins/SchemaToDefaultsPlugin');
+const {SchemaToTypeScriptPlugin} = require('openfin-service-config/plugins/SchemaToTypeScriptPlugin');
 /**
  * Import the webpack tools from openfin-service-tooling
  */
 const webpackTools = require('openfin-service-tooling').webpackTools;
 
+const schemaRoot = path.resolve(__dirname, './res/provider/config');
+const schemaOutput = path.resolve(__dirname, './gen/provider/config');
+const defaultsOutput = path.resolve(__dirname, './gen/provider/config/defaults.json');
+
+/**
+ * Generate TypeScript definition files from the config schema files.
+ *
+ * Generated code is placed inside a top-level 'gen' folder, whose structure mirrors that of
+ * the 'src', 'res' and 'test' folders.
+ */
+const schemaDefaultsPlugin = new SchemaToDefaultsPlugin({
+    outputPath: defaultsOutput,
+    input: `${schemaRoot}/fdc3-config.schema.json`
+});
+
+/**
+ * Generate TypeScript definition files from the config schema files.
+ *
+ * Generated code is placed inside a top-level 'gen' folder, whose structure mirrors that of
+ * the 'src', 'res' and 'test' folders.
+ */
```

Comment:
Should these two block comments be exactly the same? As someone who doesn't get webpack very well, I want the understand the difference between `SchemaToTypeScriptPlugin` and `SchemaToDefaultsPlugin`

Reply:
Whoops.  The plugin classes had the descriptions on them.  I've moved them into the webpack for easier reading.